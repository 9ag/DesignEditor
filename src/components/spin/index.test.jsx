
import React from 'react';
import renderer from 'react-test-renderer';

import Spin from './index';


describe("Spin component test", () => {
  it("Spin snapshot test", () => {
    const component = renderer.create(<Spin style={{ margin: `${20}px auto` }} />).toJSON();
    expect(component).toMatchSnapshot();
  });

});