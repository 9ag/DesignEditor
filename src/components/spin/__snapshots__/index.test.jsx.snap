// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spin component test Spin snapshot test 1`] = `
<div
  className="loader"
  style={
    Object {
      "margin": "20px auto",
      "width": 50,
    }
  }
>
  <svg
    className="circular"
    viewBox="25 25 50 50"
  >
    <circle
      className="path"
      cx="50"
      cy="50"
      fill="none"
      r="20"
      stroke-miterlimit="10"
      stroke-width={3}
    />
  </svg>
</div>
`;
